configs/segcache.toml

```diff
@@ -27,8 +27,8 @@ admin = "127.0.0.1:9090"
 # only valid for parquet output
 #histogram = "sparse"
 # optionally, specify the sampling interval for metrics. Input is a string
-# with the unit attached; for example "100ms" or "1s". Default to 100ms.
-#interval = "1s"
+# with the unit attached; for example "100ms" or "1s". Defaults to 1s.
+#interval = "100ms"
```

Comment:
I like to have the example values be the defaults so that when it's uncommented, the behavior doesn't change until a change is made to the value.